packages/playwright/src/executors/playwright/playwright.ts

```diff
@@ -89,11 +89,15 @@ export async function playwrightExecutor(
   });
 }
 
+const nxOptions = ['skipInstall'];
 function createArgs(opts: PlaywrightExecutorSchema): string[] {
   const args: string[] = [];
 
   for (const key in opts) {
     const value = opts[key];
+    if (nxOptions.includes(key)) {
+      continue;
+    }
```

Comment:
let's pass this as an option in the function instead and just set the defaults

```suggestion
function createArgs(opts: PlaywrightExecutorSchema, exclude = ['skipInstall']): string[] {
  const args: string[] = [];

  for (const key in opts) {
    const value = opts[key];
    if (exclude.includes(key)) {
      continue;
    }
```